libs/dh/charges/feature-prices/src/lib/drawer/charge-message/dh-charge-price-message.component.ts

```diff
@@ -14,30 +14,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { CommonModule } from '@angular/common';
 import { Component, OnDestroy, OnInit } from '@angular/core';
+import { MatDividerModule } from '@angular/material/divider';
 import {
   DhChargeMessageArchiveDataAccessStore,
   DhMarketParticipantDataAccessApiStore,
 } from '@energinet-datahub/dh/charges/data-access-api';
+import { DhMessageArchiveDataAccessBlobApiStore } from '@energinet-datahub/dh/message-archive/data-access-api';
 import {
   MarketParticipantV1Dto,
   MessageArchiveSearchCriteria,
   MessageArchiveSearchResultItemDto,
 } from '@energinet-datahub/dh/shared/domain';
-import { PushModule } from '@rx-angular/template/push';
-import { LetModule } from '@rx-angular/template/let';
-import { Subject, take, takeUntil } from 'rxjs';
-import { DhChargesPricesDrawerService } from '../dh-charges-prices-drawer.service';
 import { DhSharedUiDateTimeModule } from '@energinet-datahub/dh/shared/ui-date-time';
-import { TranslocoModule } from '@ngneat/transloco';
+import { WattBadgeComponent } from '@energinet-datahub/watt/badge';
 import { WattButtonModule } from '@energinet-datahub/watt/button';
+import { WattEmptyStateModule } from '@energinet-datahub/watt/empty-state';
 import { WattIconModule } from '@energinet-datahub/watt/icon';
-import { DhMessageArchiveDataAccessBlobApiStore } from '@energinet-datahub/dh/message-archive/data-access-api';
 import { WattSpinnerModule } from '@energinet-datahub/watt/spinner';
-import { CommonModule } from '@angular/common';
-import { WattBadgeComponent } from '@energinet-datahub/watt/badge';
-import { MatDividerModule } from '@angular/material/divider';
-import { WattEmptyStateModule } from '@energinet-datahub/watt/empty-state';
+import { TranslocoModule } from '@ngneat/transloco';
+import { LetModule } from '@rx-angular/template/let';
+import { PushModule } from '@rx-angular/template/push';
+import { Subject, take, takeUntil } from 'rxjs';
+import { DhChargesPricesDrawerService } from '../dh-charges-prices-drawer.service';
```

Comment:
question: do you have an extension that did this? Looks like the imports are sorted alphabetically :)

Reply:
This is done by VSCode with this setting in my settings.json:

` 
"editor.codeActionsOnSave": {
    "source.organizeImports": true,
    "source.sortMembers": true
  }
`

Reply:
I see. Also a good solution. So it looks like everyone has their own way of organizing imports :D 

Some just add them as they come, without order, others put them in groups, you let VS Code do it, ...  

Not sure what a common solution would look like, but that shouldn't stop the PR from merging.